src/data-sources/plugins/PluginsMapping.ts

```diff
@@ -0,0 +1,8 @@
+import Constant from './Constant/index';
```

Comment:
No, it's in use. 
Let's discuss this as I haven't found a quick yet elegant solution.
This has created a technical dept.